### DISCUSSION TITLE
Unable to Reverify Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by dotjson01** [community/community#62649] [GitHub Education]
### Hi everyone,

Unfortunately, we weren't able to approve your educational discount request for [@thestarsahil](https://github.com/thestarsahil).

What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

Attendees of this school must verify and use their academic email address.

Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher.
![Request-a-discount-GitHub-Education](https://github.com/community/community/assets/84167034/3eb77a0f-cba0-4f05-9a31-1df44c4f52b1)

### A suggestion...(optional)

Github Team , Kindly Look onto this matter 
I Shall be very thankful to you !

**Comment by HvTung04** [comment]
may I ask what did you upload?

**Comment by dotjson01** [comment]
the rejection notification

**Comment by prabhjotschugh** [comment]
You can try to upload your school's ID card (with validity mentioned on it) or your recent fee receipt as a proof that you are still a student. May be the document that you uploaded was very old.
Hope it helps!

**Comment by dotjson01** [comment]
Bot is rejecting on first condition 
that's mail 
on 2022 My mail accepted but on 2023 isn't accepting , i am reverifying that same email 
Now as per according to Gtihub Student have a access till end sem

**Comment by prabhjotschugh** [comment]
So now you are not a student? or have you completed your course?

**Comment by dotjson01** [comment]
i am still a 3rd year student
i applied when i was 2nd year

**Comment by prabhjotschugh** [comment]
Then you are still eligible for the benefits. Kindly visit this page (https://education.github.com/benefits) and reverify.

**Reply by prabhjotschugh** [reply]
make sure you include a valid supporting document.

**Reply by dotjson01** [reply]
i am submitting my calender academic

**Reply by prabhjotschugh** [reply]
No, you can submit your college's ID card (with its validity visible) or your most recent fee receipt. Academic calender would not work.

**Reply by dotjson01** [reply]
I also used that ...but nothing happening

**Reply by GhostOf0days** [reply]
Try using a screenshot of your cost of attendance bill if it has your school logo on it.

**Reply by dotjson01** [reply]
i think the major part rejection is mail id 
I mean in just 30sec i got mail that i am rejected

**Comment by Spixmacaw** [comment]
Hi everyone,
Unfortunately, we weren't able to approve your educational discount request for [@thestarsahil](https://github.com/thestarsahil).

What happened?
The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

Attendees of this school must verify and use their academic email address.

Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher.

![image]("https://user-images.githubusercontent.com/84167034/258356937-3eb77a0f-cba0-4f05-9a31-1df44c4f52b1.png")

**Reply by dotjson01** [reply]
? same copied

**Comment by Lodene** [comment]
Hi, have you find any solution ?
